common/djangoapps/auth_exchange/views.py

```diff
@@ -0,0 +1,78 @@
+# pylint: disable=abstract-method
+"""
+Views to support third-party to first-party OAuth 2.0 access token exchange
+"""
+from django.conf import settings
+from django.contrib.auth import login
+import django.contrib.auth as auth
+from django.http import HttpResponse
+from django.utils.decorators import method_decorator
+from django.views.decorators.csrf import csrf_exempt
+from provider import constants
+from provider.oauth2.views import AccessTokenView as AccessTokenView
+from rest_framework import permissions
+from rest_framework.views import APIView
+import social.apps.django_app.utils as social_utils
+
+from auth_exchange.forms import AccessTokenExchangeForm
+from openedx.core.lib.api.authentication import OAuth2AuthenticationAllowInactiveUser
+
+
+class AccessTokenExchangeView(AccessTokenView):
+    """View for access token exchange"""
```

Comment:
With the module docstring needing to change to be more general, this should be more specific.